src/20240720_AaveV3EthereumLido_LidoEthereumInstanceActivation/AaveV3EthereumLido_LidoEthereumInstanceActivation_20240720.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3EthereumLido} from 'aave-address-book/AaveV3EthereumLido.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereumLido} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereumLido.sol';
+import {EngineFlags} from 'aave-v3-periphery/v3-config-engine/EngineFlags.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-periphery/v3-config-engine/IAaveV3ConfigEngine.sol';
+
+/**
+ * @title Lido Ethereum Instance Activation
+ * @author Catapulta @catapulta_sh
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0x9403b7f704ade0d6510591e4846fd85c84b19d134c0814511af914751ecfad5d
+ * - Discussion: https://governance.aave.com/t/arfc-deploy-a-lido-aave-v3-instance/18047
+ */
+contract AaveV3EthereumLido_LidoEthereumInstanceActivation_20240720 is AaveV3PayloadEthereumLido {
+  using SafeERC20 for IERC20;
+
+  address public constant wstETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+  uint256 public constant wstETH_SEED_AMOUNT = 1e17;
+  address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+  uint256 public constant WETH_SEED_AMOUNT = 1e17;
+
+  function _postExecute() internal override {
```

Comment:
@kartojal we should fix the incentives controller on the addresses provider, maybe better in `_preExecute()`

Reply:
Also I think it would be a good idea to re-use the addresses-provider-registry